### DISCUSSION TITLE
A virus of type a.gray.BulimiaTGrn.f

### DISCUSSION THREAD

**Discussion by XuanTian-LTD** [rustdesk/rustdesk#8551] [Q&A]
![Screenshot_2024-06-30-23-02-31-628_com miui securitycenter-edit](https://github.com/rustdesk/rustdesk/assets/104124862/7ceb858a-0a3c-4a89-88c0-dfda3e2f461d)
Why was a virus found?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/2485
